test/samples_test.cpp

```diff
@@ -117,7 +121,6 @@ TEST(Samples, Scallop) {
 
   scallop = scallop.Refine(50).CalculateCurvature(-1, 0).SetProperties(
       3, colorCurvature);
-  CheckNormals(scallop);
```

Comment:
do we still want to check for normals now? iirc the normal check can be useful when we have degenerate triangles?

Reply:
Honestly it wasn't checking much anyway and it was mostly just there because normals were part of our output. We still have a separate degenerate triangles check - we can always add that to more tests if we want.